### PR TITLE
WIP: Add styled components

### PR DESCRIPTION
**Pull request by rolfea**
**- Summary**
This adds the styled-components library to the project and starts the process of replacing the css with individual styled-components.

This solves the lack of styled components. Should make the maintenance of the styling more idiomatic.
**- Test plan**
None at the moment. I could look into setting up some selenium testing to verify the elements are working as expected.

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```
Comment (bdougie) [inline .vscode/settings.json]: Is this needed?

File: src/App.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (bdougie) [inline src/App.js]: Be sure to remove when linting is all fixed.

**Comment by bdougie** [comment]
@rolfea can you make this an incremental change to just add the Button styling. I think making incremental PR's is the way to go. 

When you have removed the comments cleaned up lint, it would be ideal to merge this.

As far as testing. I would like to address some sort of unit testing structure first before approach integration or acceptance testing through selenium

**Comment by rolfea** [comment]
@bdougie sounds good - I will make this a more incremental change and clean up the linting  
we should chat about what the best approach for unit testing would be

**Comment by bdougie** [comment]
Going to close for now. I am updating to Style components 2.0

**Pull request closed without merging**